### ISSUE TITLE
Cannot use some hotkeys to choose tank to fill in a vehicle

### ISSUE THREAD

**Issue by Zetsukaze** [CleverRaven/Cataclysm-DDA#26467]
**Describe the bug**  
Some hotkeys can be used to select tanks, while others cannot. I have observed this on previous builds but only confirmed this for the current build.

**To Reproduce**  
1. Have a vehicle with a few tanks
2. Examine the vehicle to bring up the vehicle screen, press [f] to refill a tank
3. Press [g] or [h] and observe nothing happens
4. Press [c] or [d] and observe that the screen to choose which liquid to fill appears

**Expected behavior**  
All the hotkeys shown work

**Screenshots**  
![refill1](https://user-images.githubusercontent.com/1698993/47782590-893bb280-dd3b-11e8-966c-726bff8ea114.PNG)

**Versions and configuration(please complete the following information):**
Windows 10, Build 8097

**Comment by kevingranade** [comment]
This is handled here
https://github.com/CleverRaven/Cataclysm-DDA/blob/fffed0be4b064fcddf1c176978aa847952046ed9/src/veh_interact.cpp#L1246
I'm guessing this little lambda
https://github.com/CleverRaven/Cataclysm-DDA/blob/fffed0be4b064fcddf1c176978aa847952046ed9/src/veh_interact.cpp#L1272-1278
needs to skip over reserved keybindings.

**Comment by chrimchim** [comment]
Hello, I am an amateur coder looking to work on the project with my only experience being the free Codecademy C++ course I completed to try to learn more about it. I've been looking at this problem for a couple days and I can't see an easy way to tell which keys are bound and which aren't. I tried looking at the main inventory's code for a solution but that is in many ways more complicated and uses a lot of dependencies. I would greatly appreciate any info on where to look for solutions or where this problem is fixed in other parts of the game as I continue to figure this out since.
